Possible missing dependency on babel-runtime

**I'm submitting a bug report**

**Webpack version:**
3.11.0

**Webpack Karma version**:
1.7.x

**Karma version**:
2.0.11

**Please tell us about your environment:**
OSX

**Current behavior:**

```
$ karma start
25 02 2018 19:56:08.367:ERROR [config]: Error in config file!
 { Error: Cannot find module 'babel-runtime/helpers/toConsumableArray'
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/cthielen/projects/uirouter/sticky-states/node_modules/karma-webpack/lib/karma-webpack.js:3:27)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/cthielen/projects/uirouter/sticky-states/node_modules/karma-webpack/lib/index.js:7:18)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32) code: 'MODULE_NOT_FOUND' }
```


* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem along with a gist/jsbin of your webpack configuration.**

```
git clone https://github.com/ui-router/sticky-states.git
cd sticky-states
git checkout karma-webpack
yarn
yarn add --dev karma-webpack@2.0.11
yarn test
```

Adding `babel-runtime` as a dep fixes the issue.  Guessing this was due to https://github.com/webpack-contrib/karma-webpack/commit/6d28c5cf1246e1659a7553c18af6f43de84a7db5.  2.0.9 was the last version of karma-webpack that worked without babel-runtime.

--- a/package.json
+++ b/package.json
@@ -41,6 +41,7 @@
     "babel-polyfill": "^6.0.0",
     "babel-preset-env": "^1.0.0",
     "babel-register": "^6.0.0",
+    "babel-runtime": "^6.0.0",
     "chai": "^4.0.0",
     "chai-as-promised": "^7.0.0",
     "cross-env": "^5.0.0",

